packages/core/src/electron-browser/menu/electron-main-menu-factory.ts

```diff
@@ -76,14 +76,14 @@ export class ElectronMainMenuFactory {
 
     createContextMenu(menuPath: MenuPath, args?: any[]): Electron.Menu {
         const menuModel = this.menuProvider.getMenu(menuPath);
-        const template = this.fillMenuTemplate([], menuModel, args);
-
+        const template = this.fillMenuTemplate([], menuModel, args, true);
         return electron.remote.Menu.buildFromTemplate(template);
     }
 
     protected fillMenuTemplate(items: Electron.MenuItemConstructorOptions[],
         menuModel: CompositeMenuNode,
-        args: any[] = []
+        args: any[] = [],
+        isContextMenu?: boolean,
```

Comment:
(nit) for last optional it is better to use an object, it is easy to extend without breaking later